src/averageAges.js

```diff
@@ -37,7 +45,14 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people.filter(woman => woman.sex
+      === 'f' && people.some(person => person.mother === woman.name))
+    : people.filter(woman => woman.sex === 'f');
```

Comment:
You could use the same approach as in the calculateMenAverageAge function.  Use a ternary operator inside the filter fcuntion